e2e/server_test.go

```diff
@@ -110,8 +110,8 @@ func TestProxyServerWithLargeBlob(t *testing.T) {
 		URL: ts.Address(),
 	}
 	daClient := client.New(cfg)
-	//  4MB blob
-	testPreimage := []byte(e2e.RandString(4_000_000))
+	//  16MB blob
+	testPreimage := []byte(e2e.RandString(16_000_000))
```

Comment:
eigenda accounting uses 2^24, 16777216

Reply:
right - the default codec client will pad to the next power of two when doing dispersal